react-frontend/src/authentication/AdditionalDetailsPage.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import { useNavigate, Link } from 'react-router-dom';
 import logoImage from '../assets/haggle-horizontal.png';
+import "./AuthenticationStyling.css"
```

Comment:
I like that you made this a authentication styling css file. way better naming good job.

Reply:
I agree